app/views/coronavirus_form/check_your_answers/sections/_accommodation.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= render "govuk_publishing_components/components/summary_list", {
+  title: t("coronavirus_form.check_your_answers.sections.accommodation.title"),
+  heading_level: 2,
+  heading_size: "l",
+  items: question_items("accommodation"),
+  edit: {
+    href: "accommodation?change-answer",
+  },
+} %>
```

Comment:
Again the outer curly brace is not needed

Reply:
Hi @reggieb, sorry you are incorrect - the braces are required.  The page throws a syntax error without them.

Reply:
Fair enough!